packages/astro/src/@types/astro.ts

```diff
@@ -1351,6 +1351,25 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		optimizeHoistedScript?: boolean;
+
+		/**
+		 * @docs
+		 * @name experimental.devOverlay
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 3.4.0
+		 * @description
+		 * Enable a dev overlay in development mode. This overlay allows you to inspect your page islands, see helpful audits on performance and accessibility, and more.
+		 *
+		 * ```js
+		 * {
+		 * 	experimental: {
+		 * 		devOverlay: true,
+		 * 	},
+		 * }
+		 * ```
+		 */
+		devOverlay?: boolean;
```

Comment:
Sorry in advance... I'm going to be that person. How attached are we to the name "Dev Overlay"? I feel like I've only ever seen this kind of feature referred to as "___ DevTools" or "___ Inspector". Those might be more immediately understandable names for most users.

Reply:
My argument against it is that Astro is a dev tool, Prettier is a dev tool, VS Code is a dev tool etc. The dev overlay is an overlay that's shown in dev, it's what it is!

(Anecdotally, I always search for "Nuxt Dev Overlay" and can never find it, which annoys me a lot)

Reply:
I like dev overlay better as well. Any name other than "devtools" since, as @Princesseuh mentioned, VScode is a dev tool.

Reply:
OK, that's a fair justification! I still personally like "Inspector," but if everyone else is happy with Overlay, I'm happy!